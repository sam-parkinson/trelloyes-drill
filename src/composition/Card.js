import React from 'react';
import './Card.css'

function Card(props) {
  console.log(props);
  return null;
}

export default Card;