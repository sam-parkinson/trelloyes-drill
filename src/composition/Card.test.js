import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Card from './Card';

describe('List component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Card />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the UI as expected', () => {
    const tree = renderer
      .create(<Card className='Card' key='0' title='test' content='test'/>)
      .toJSON();
    expect(tree).toMatchSnapshot();  
  });

});