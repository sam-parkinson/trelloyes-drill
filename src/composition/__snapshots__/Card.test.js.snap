// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List component renders the UI as expected 1`] = `
<div
  className="Card"
>
  <h3>
    test
  </h3>
  <p>
    test
  </p>
</div>
`;
